services/TokenGetAccountNftInfos.proto

```diff
@@ -32,6 +32,18 @@ import "ResponseHeader.proto";
 
 /* Applicable only to tokens of type NON_FUNGIBLE_UNIQUE. Gets info on NFTs N through M owned by the specified accountId.
  * Example: If Account A owns 5 NFTs (might be of different Token Entity), having start=0 and end=5 will return all of the NFTs
+  *
+ * INVALID_QUERY_RANGE response code will be returned if:
+ * 1) Start > End
+ * 2) Start and End indices are non-positive
+ * 3) Start and End indices are out of boundaries for the retrieved nft list
+ * 4) The range between Start and End is bigger than the global dynamic property for maximum query range
```

Comment:
Please use lower-case start and end to match the fields. `start` is inclusive and `end` is exclusive so:
1) start >= end
2) should split into 2 different bullet points: start is negative (because start could be zero), and end is non-positive